.github/workflows/linux_build_tests.yaml

```diff
@@ -0,0 +1,25 @@
+name: linux builds
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Packages
+      run: |
+        sudo apt-get update
+        sudo apt-get install -yqq gcc g++ gfortran lua5.1 lua5.1-dev make
+    - name: Build IREP
+      run: |
+        make CPPFLAGS=-I/usr/include/lua5.1
+    - name: Build and Run Examples
+      run: |
+        make LUA_INCLUDE=-I/usr/include/lua5.1 LUA_LIBRARIES="-L/usr/lib/x86_64-linux-gnu -llua5.1"
```

Comment:
Does this run the examples? or were you planning to add that in the future?

It seems to me that you'd have to `cd` into the `examples` directory? And it doesn't look like it is actually building/running the examples: 
https://github.com/LLNL/irep/runs/2049666979?check_suite_focus=true#step:5:4

Reply:
(I could be missing something though...)

Reply:
oh nice catch.  I think I deleted the `test` target when I added the lua libs.

Reply:
Ok this is fixed -- I also split it out into 3 make calls that show up as steps in the action.